plugins/typescript/src/templates/fetcher.ts

```diff
@@ -137,6 +137,6 @@ const resolveUrl = (
 ) => {
   let query = new URLSearchParams(queryParams).toString();
   if (query) query = \`?\${query}\`;
-  return url.replace(/\\{\\w*\\}/g, (key) => pathParams[key.slice(1, -1)]) + query;
+  return url.replace(/\\{\\w*\\}/g, (key) => pathParams[key.slice(1, -1)] || key) + query;
```

Comment:
Even if this is just for the compiler, I think the expected behaviour is to fallback on an empty value instead of the key

```suggestion
  return url.replace(/\\{\\w*\\}/g, (key) => pathParams[key.slice(1, -1)] ?? "") + query;
```